apps/console/src/features/identity-providers/data/identity-provider-templates/templates/saml-identity-provider/saml-idp-wizard-file-based-help.tsx

```diff
@@ -40,6 +41,13 @@ const SAMLIdPWizardFileBasedHelp: FunctionComponent<Props> = (props: Props): Rea
 
     return (
         <div data-testid={ testId }>
+            <Heading as="h5">Service provider entity ID</Heading>
```

Comment:
Lets add i18n for newly added strings.

Reply:
Not exactly new. Taken from manual configuration help-panel.